docs/Advanced-Usage/Manager/Manager-Configuration-Files.rst

```diff
@@ -377,8 +377,7 @@ you made up). Such a section must contain the following fields:
 """""""""""""""""""""""""""""
 
 This specifies the basic target design that will be built. Unless you
-are defining a custom system, this should either be ``FireSim``, for
-systems with a NIC, or ``FireSimNoNIC``, for systems without a NIC. These
+are defining a custom system, this should either ``FireSim``. These
 are defined in ``firesim/sim/src/main/scala/firesim/Targets.scala``. We
```

Comment:
@jerryz123 is this still correct?

Reply:
(this is referring to

are defined in ``firesim/sim/src/main/scala/firesim/Targets.scala``.

)

Reply:
These docs are quite stale, they predate the firechip->chipyard merge.

Reply:
can we just remove that sentence with the path then for now